scripts/Linux-Run.sh

```
@@ -34,10 +48,20 @@ if [ -n "${BUILD_CONFIG+set}" ]
-export VULKAN_SDK=$(realpath Core/vendor/VulkanSDK/x86_64)
+
+CONFIG=$(echo "$BUILD_CONFIG" | tr '[:upper:]' '[:lower:]')
+
+# Linux-Build.sh is idempotent (make only rebuilds what changed), so this catches forgotten
+# recompiles without meaningfully slowing down a launch where nothing moved. BUILD_CONFIG is
+# already exported above, so it won't re-prompt.
+if [ -z "${LUX_SKIP_BUILD+set}" ]; then
+	"$LUX_DIR/scripts/Linux-Build.sh"
+fi
+
+export VULKAN_SDK=$(realpath "$LUX_DIR/Core/vendor/VulkanSDK/x86_64")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)